### PR TITLE
Basic config and setup

### PR DESCRIPTION
**Pull request by DebyGrey**
In this project, I:

- Created React App.
- Installed React Redux, Redux Logger, and React Route
- Setup linter checkers
- Downloaded the free image for the app logo from the flat-icon website
- Created routes and view components: rockets, missions, my profile.
- Used <NavLink /> for the page navigation links 
- Created directories for all Redux state slice files: rockets, and missions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +115,10 @@ You can deploy this project:
 - Twitter: [@twitterhandle](https://twitter.com/Deby_grey)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/deborah-fashoro)
 
+👤 **Martin Karugaba**
+
+- GitHub: [@githubhandle](https://github.com/martinkarugaba)
+
```
Comment (martinkarugaba) [inline README.md:121]: Kindly add my linkedin and my twitter to here.
[My linkedin profile](https://www.linkedin.com/in/martinkarugaba/)
[My twitter profile](https://twitter.com/martin_karugaba)

**Comment by martinkarugaba** [comment]
Well done @DebyGrey 

:heavy_check_mark: Created React App.
:heavy_check_mark: Installed React Redux, Redux Logger, and React Route.
:heavy_check_mark: Linters set up correctly.
:heavy_check_mark: Downloaded the free image for the app logo from the flat-icon website.
:heavy_check_mark: Created routes and view components: rockets, missions, my profile.
:heavy_check_mark: Created directories for all Redux state slice files: rockets, and missions.

:orange_circle: Required changes
Check the inline comment below for required changes

File: README.md (hunk 1 above)
Comment (martinkarugaba) [inline README.md:121]: Kindly add my linkedin and twitter profile here;
[My linkedin profile](https://www.linkedin.com/in/martinkarugaba/)
[My twitter profile](https://twitter.com/martin_karugaba)

Comment (DebyGrey) [reply README.md]: @martinkarugaba Fixed

**Review by martinkarugaba — APPROVED** [review]
Great work @DebyGrey

**Pull request merged**